app/packages/core/src/components/Modal/TooltipInfo.tsx

```diff
@@ -170,7 +174,7 @@ const AttrInfo = ({ label, children = null }) => {
   );
 
   const other = entries.filter(
-    ([name]) => !["label", "confidence"].includes(name)
+    ([name]) => !["label", "confidence", "bounding_box"].includes(name)
```

Comment:
I think this will need to be generalized based on the `Label` type:
- `Detection`: don't include `bounding_box`/`mask` in tooltip
- `Classification`: don't include `logits` in tooltip
- `Keypoint`: don't include `points` in tooltip
- `Polyline`: don't include `points` in tooltip
- `Segmentation`: don't include `mask` in tooltip
- `Heatmap`: don't include `map` in tooltip

Based on the following rules:
- `bounding_box`/`points` are excluded because this data is already rendered spatially.
- `mask`/`map`/`logits` are excluded because they are binary (array) types, which aren't suitable for rendering (this may not need to be explicitly added, if binary field values are already implicitly excluded; I didn't check the implementation carefully)

Reply:
@brimoor Does `Regression` type also have some label that we do not want to show?